dot/errors.go

```diff
@@ -12,3 +12,6 @@ var ErrNoKeysProvided = errors.New("no keys provided for authority node")
 
 // ErrInvalidKeystoreType when trying to create a service with the wrong keystore type
 var ErrInvalidKeystoreType = errors.New("invalid keystore type")
+
+// ErrWasmInterpreterName is returned when trying to create a service with the wrong wasm interpreter name
```

Comment:
nit no need the linter is configured not to complain about it for errors, since it should be self explanatory

```suggestion
```

Reply:
Removed.